rubyparser.h

```diff
@@ -34,6 +34,18 @@
 #endif
 #endif
 
+#if defined(__GNUC__)
+# if defined(__MINGW32__) || defined(__MINGW64__)
```

Comment:
Simple check for `__MINGW_PRINTF_FORMAT` doesn't work?

```suggestion
# if defined(__MINGW_PRINTF_FORMAT)
```

Reply:
Gosh, you are right!